### PR TITLE
run job for all supported APIs

### PR DESCRIPTION
**Pull request by greenEkatherine**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -6,77 +6,95 @@
...
-        for i in $(seq 1 1 20)
-        do
-          screenrecord --bugreport --verbose --time-limit=180 /mnt/sdcard/DCIM/test"${i}".mp4
-        done
-      ''' > $(Build.ArtifactStagingDirectory)/video.log &
-      echo $! > video.pid
-      echo "Video Recorder Started with Process ID: $(cat video.pid)"
-    displayName: 'Start Video Record and Logcat'
+        - bash: |
+            ./gradlew connectedDebugAndroidTest --info
```
Comment (omen273) [inline azure-pipelines.yml:76]: Why Debug?

**Review by omen273 — CHANGES_REQUESTED** [review]
In my opinion, it is better to make clean up before every test. Otherwise tests can affect each other and it is reall difficullt to invistigate the reason of a problem.

**Comment by greenEkatherine** [comment]
> In my opinion, it is better to make clean up before every test. Otherwise tests can affect each other and it is reall difficullt to invistigate the reason of a problem.

It will be done in separate task

File: azure-pipelines.yml (hunk 1 above)
Comment (greenEkatherine) [reply azure-pipelines.yml]: I've tried release but it seems to require signing:

`Unable to install /Users/runner/work/1/s/app/build/outputs/apk/release/app-release-unsigned.apk
com.android.ddmlib.InstallException: INSTALL_PARSE_FAILED_NO_CERTIFICATES: Failed to collect certificates from /data/app/vmdl180142812.tmp/base.apk: Attempt to get length of null array
`

[Link to failed build](https://dev.azure.com/warpedomen/warpedomen/_build/results?buildId=38&view=logs&j=aa2fd561-a338-5537-45ee-49ae22aea958&t=e99b6747-5a34-5948-7b31-f1617fea6ef1&l=34065)

Anyway it doesn't set up in the pipeline, we should add build type in gradle - I've opened issue for it https://github.com/omen273/crosswordApp/issues/52

**Review by omen273 — APPROVED** [review]

**Pull request merged**